src/test/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelperTest.java

```diff
@@ -41,96 +43,82 @@
     private static final DateTime TEST_START_DATETIME = new DateTime();
     private static final DateTime TEST_END_DATETIME = new DateTime();
 
-    private static final ObjectNode TEST_REPORT_DATA = JsonNodeFactory.instance.objectNode();
-    static {
-        TEST_REPORT_DATA.put("field1", "test");
-    }
-    private static final ReportData TEST_REPORT = new ReportData(TEST_START_DATETIME.toLocalDate(), TEST_REPORT_DATA);
+    private static final Map<String, String> TEST_REPORT_DATA = ImmutableMap.of("field1", "test");
+    private static final ReportData TEST_REPORT = new ReportData().date(TEST_START_DATETIME.toLocalDate())
+            .data(TEST_REPORT_DATA);
 
-    private static final StudySummary TEST_STUDY_SUMMARY;
-    static {
-        TEST_STUDY_SUMMARY = new StudySummary();
-        TEST_STUDY_SUMMARY.setIdentifier(TEST_STUDY_ID);
-        TEST_STUDY_SUMMARY.setName(TEST_STUDY_ID);
-    }
-    private final ResourceList<StudySummary> TEST_STUDY_SUMMARY_LIST = new ResourceList<>(Arrays.asList(TEST_STUDY_SUMMARY), 1);
+    private static final Study TEST_STUDY_SUMMARY = new Study().identifier(TEST_STUDY_ID).name(TEST_STUDY_ID);
 
     private Upload testUpload;
-    private ResourceList<Upload> testUploads;
-
-    private static BridgeHelper setupBridgeHelperWithSession(Session session) {
-        // Spy bridge helper, because signIn() statically calls ClientProvider.signIn()
-        BridgeHelper bridgeHelper = spy(new BridgeHelper());
-        doReturn(session).when(bridgeHelper).signIn();
-        return bridgeHelper;
-    }
 
     @BeforeClass
     public void setup() throws IOException {
-        testUpload = BridgeUtils.getMapper().readValue(json, Upload.class);
-        ResourceList<Upload> TEST_UPLOADS = new ResourceList<>(Arrays.asList(testUpload), 1);
+        testUpload = RestUtils.GSON.fromJson(json, Upload.class);
```

Comment:
To add a bit more color, we were originally using Jackson for both Bridge server and the Java SDK, but the Java SDK ended up migrating to Gson* when we redesigned it. If there is a future desire, our libraries/dependencies allow for us to switch to Jackson or support both Gson and Jackson.

* I'm trying to remember the details of why we decided it was worth incurring overhead of Gson when the team already is familiar with Jackson. I do remember that ResearchStack (the main use case for the SDK) uses Gson, and Swagger codegen does as well (but may support Jackson too?)